packages/react/carbon.yml

```diff
@@ -445,6 +460,20 @@ assets:
         name: Storybook
         action: link
         url: https://react.carbondesignsystem.com/?path=/docs/components-search
+    section:
```

Comment:
Per https://next.carbondesignsystem.com/contributing/schema#stable-version, if you install this VS Code extension: https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml, there appear to be some YAML indentation errors.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/1691245/185680347-35c7bbba-2254-4787-affe-4e2967fe663f.png">

Reply:
Thanks! Great extension 🎉 

Fixed